### PR TITLE
WIP: Choosing a CMS platform

### PR DESCRIPTION
**Pull request by cromwellryan**
- [ ] When to recommend Drupal
- [ ] When to recommend CraftCMS

### CODE REVIEW

File: platforms/drupal.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+### Use Drupal Coding Standards
+
+When writing custom code, follow [the Drupal Coding Standards](https://www.drupal.org/docs/develop/standards), as documented on Drupal.org.
+
+Several tools exist for linting against these standards including [phpcs+coder](https://www.drupal.org/project/coder), [Drupal's ESLint config](https://www.drupal.org/docs/develop/standards/javascript/eslint-settings), and [CSSComb](https://www.drupal.org/docs/develop/standards/css/csscomb-settings-for-drupal-css-formatting-and-sort-tool).
+
+### Stay on top of security updates
+
+Drupal core security announcements are posted at https://www.drupal.org/security, which you can subscribe to via email or RSS. Composer makes Drupal core updates easy so there's no reason to get behind.
```
Comment (cromwellryan) [inline platforms/drupal.md]: Can you use `composer` outdated to check this? Do you know of any services like SourceClear, Snyk, etc that support composer?

Comment (nrambeck) [inline platforms/drupal.md]: Composer itself does not track security vulnerabilities in projects, but SensioLabs maintains a database of vulnerabilities that is contributed to voluntarily. 
https://security.sensiolabs.org/database

The Drupal project has a very robust security review and reporting process, not only for core code, but also for contributed modules. The reporting section of the Drupal Admin UI will show any security updates that are required and these warnings can be emailed to an administrator. The Drush CLI tool will do the same and can be leveraged during CI builds or other automated processes to report vulnerabilities.

**Comment by catheraaine** [comment]
@kaseybon here is a great starter for your updates

**Comment by catheraaine** [comment]
See #108

**Pull request merged**